CHANGELOG.develop

```diff
@@ -1463,6 +1463,42 @@ Other:
   - Removed ~SPC m h g~ for =grimoire= as it is deprecated (thanks to Sam Hedin)
   - ~C-return~ to =cider-repl-newline-and-indent= in REPL evil insert state
     (thanks to Gia Thanh Vuong)
+  - changed clear repl buffer of evaluation to match terminal clear key
+    ~SPC s s l~ 'cider-repl-clear-buffer
+    ~SPC s s L~ 'cider-find-and-clear-repl-output
+    (thanks to John Stevenson)
+  - Add sesman session management keybindings to ~SPC m c~
+    ~SPC m c b~ 'sesman-browser
+    ~SPC m c i~ 'sesman-info
+    ~SPC m c g~ 'sesman-goto
+    ~SPC m c l b~ 'sesman-link-buffer
+    ~SPC m c l d~ 'sesman-link-directory
+    ~SPC m c l p~ 'sesman-link-project
+    ~SPC m c l u~ 'sesman-unlink
+    ~SPC m s q q~ 'sesman-quit
+    ~SPC m s q r~ 'sesman-restart
+    ~SPC m c s~ 'sesman-start
+    ~SPC m c S j~ 'cider-connect-sibling-clj
+    ~SPC m c S s~ 'cider-connect-sibling-cljs
+    (thanks to John Stevenson)
+  - updated repl shortcut to use sesman-start wrapper, selecting any repl type
+    ~SPC m '~ 'sesman-start
+  - changed toggle between source and repl to match key for toggle between code
+    and test
+    ~SPC m s a~ (if (eq m 'cider-repl-mode)
+                     'cider-switch-to-last-clojure-buffer
+                   'cider-switch-to-repl-buffer)
+    (thanks to John Stevenson)
+  - added formatting command not previously included
+    ~SPC m f d~ 'cider-format-defun
+    ~SPC m f e b~ 'cider-format-buffer
```

Comment:
formatting should be:

`,==` - format buffer
`,=r` - format region.

Reply:
Oh yes, something else that has not been consistent for a long time.  I will update these keybindings as well.  Thanks for spotting that.

https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#code-formatting

John.

Reply:
Formatting menu updated to use the Spacemacs convention, `=` instead of `f`